freshsales-suite/contact-kyc/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "platform-version": "2.2",
+  "product": {
+    "freshworks_crm": {
+      "location": {
+        "contact_entity_menu": {
+          "url": "contact_entity_menu.html",
+          "icon": "images/icon.svg"
+        },
+        "full_page_app": {
+          "url": "kyc_pending_fullpage_app.html",
+          "icon": "images/icon.svg"
+        }
+      }
+    },
+    "freshsales": {
+      "location": {
+        "contact_entity_menu": {
+          "url": "contact_entity_menu.html",
+          "icon": "images/icon.svg"
+        },
+        "full_page_app": {
+          "url": "kyc_pending_fullpage_app.html",
+          "icon": "images/icon.svg"
+        }
+      }
+    }
+  },
+  "whitelisted-domains": [
+    "https://*.freshworks.com"
```

Comment:
This should probably be `"*.myfreshworks.com"`. Or, you can use template substitution for the `fcrm_domain` iparam like this: `<%= iparams.fcrm_domain %>`

Reply:
Changed to the URL from iparam - `<%= iparam.fcrm_domain %>`